docs/articles/blog/2017-6-00-testcafe-v0-16-0-released.md

```diff
@@ -0,0 +1,35 @@
+---
+layout: post
+title: TestCafe v0.16.0 Released
+permalink: /blog/:title.html
+---
+# TestCafe v0.16.0 Released
+
+TypeScript support, Chrome specific features (headless and device emulation) and numerous bug fixes.
+
+## Enhancements
+
+### ⚙ TypeScript support ([#408](https://github.com/DevExpress/testcafe/issues/408))
+
+### ⚙ Support running in Chrome in headless mode and in device emulation ([#1417](https://github.com/DevExpress/testcafe/issues/1417))
+
+### ⚙ Support HTML5 Drag and Drop ([#897](https://github.com/DevExpress/testcafe/issues/897))
+
+### ⚙ Fixed URL for opening remote browsers ([#1476](https://github.com/DevExpress/testcafe/issues/1476))
+
+### ⚙ No TestCafe UI on screenshots created during testing ([#1357](https://github.com/DevExpress/testcafe/issues/1357))
```

Comment:
It is more bug than enhancements. So, I propose to move this to 'Bug Fixes' section